targets/ChibiOS/_WiFi/inventek/es_wifi.c

```
@@ -0,0 +1,2746 @@
+	}
+  UNLOCK_WIFI();
+  return ret;
+}
+
+/**
+  * @brief  Wait for a client connection to the Server.
+  * @param  Obj: pointer to module handle
+  * @param  conn: pointer to the connection structure
+  * @retval Operation Status.
+  */
+ES_WIFI_Status_t  ES_WIFI_WaitServerConnection(ES_WIFIObject_t *Obj,uint32_t timeout,ES_WIFI_Conn_t *conn)
```

Cppcheck (2.21.0)

[style] The function 'ES_WIFI_WaitServerConnection' is never used.

(unusedFunction)